optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -8,22 +8,19 @@
 :experimental:
 
 
-Constraint streams are a Functional Programming form of incremental score calculation in plain Java that is easy to
-read, write and debug.
-The API should feel familiar if you've worked with Java 8 Streams or SQL.
+Constraint streams are a functional programming form of incremental score calculation in plain Java that are easy to read, write, and debug. In OptaPlanner, contraint streams are implemented thorugh the `ConstraintStreams` API.This API should feel familiar if you have worked with Java 8 Streams or SQL.
 
 [WARNING]
 ====
-The ConstraintStreams/ConstraintProvider API is an ongoing project.
-It works but it has many API gaps.
-Therefore, it is not rich enough yet to handle complex constraints.
+The `ConstraintStreams/ConstraintProvider` API is an ongoing project. It works but it has many API gaps. Therefore, it is not yet rich enough to handle complex constraints.
 <<explainingTheScore,Constraint justifications>> may not function properly.
+//COMMENT: Lukas, what specifically  might not function.  This link is to section 5.6, Explaining the score: which constraints are broken?  Or maybe we don't need to worry about this based on your comment below?
 ====
 
 // TODO Remove the previous sentence when PLANNER-1709 is fixed.
 
-To use constraint streams in your project, implement the `ConstraintProvider` interface and use the following in your
-solver config:
+To use constraint streams in your project, implement the `ConstraintProvider` interface and inlcude the following code in your solver configuration:
+//COMMENET: Is adding the code how you implement `ConstraintProvider` or do I need to do both: Implement `ConstraintProvider` and add the code to my solve configuration?
```

Comment:
"COMMENET": if it means comment, then the "//" already imply that.
Answer: You need both: Implement a class that implements ConstraintProvider and add it in the solver configuration.

Can we move this change into a separate PR because this is a cross-cutting all-files PR?
(I don't mind if small PR's do multiple changes, but cross-cutting all-files PR I prefer isolated to avoid missing a change).

Reply:
My bad. I have reinstated the original constraint streams file and check for Planner.